mkfs/mkfs.c

```diff
@@ -622,23 +622,24 @@ int main(int argc, char *argv[])
 
 	ret = exfat_build_mkfs_info(&bd, &ui);
 	if (ret)
-		goto out;
+		goto close;
 
 	ret = exfat_zero_out_disk(&bd, &ui);
 	if (ret)
-		goto out;
+		goto close;
 
 	ret = make_exfat(&bd, &ui);
 	if (ret)
-		goto out;
+		goto close;
 
 	exfat_info("Synchronizing...\n");
 	ret = fsync(bd.dev_fd);
+close:
+	close(bd.dev_fd);
```

Comment:
namespaces between functions and goto labels are different, but "close" label seems to be confused. What do you think of changing "close" to "out_close"?